app/javascript/packs/application/components/StaffUserDashboard.js

```diff
@@ -1,37 +1,26 @@
 import React, { Component } from "react";
+import CreateTourForm from "./CreateTourForm";
 
 class StaffUserDashboard extends Component {
-  state = {
-    username: "",
-    id: "",
-    cultural_center: {
-      name: "",
-      id: ""
-    }
-  };
-
   async componentDidMount() {
     const { ok, data } = await this.props.onAuthenticate();
     if (!ok) {
       throw new Error("Not Authenticated!");
-    } else {
-      this.setState({
-        username: this.props.currentStaffUser.username,
-        id: this.props.currentStaffUser.id,
-        cultural_center: {
-          name: this.props.currentStaffUser.cultural_center.name,
-          id: this.props.currentStaffUser.cultural_center.id
-        }
-      });
     }
   }
 
   render() {
+    const { username, cultural_center, id } = this.props.currentStaffUser;
     return (
       <div>
         <h1>Staff Dashboard</h1>
-        <h3>Current user: {this.state.username}</h3>
-        <h3>{this.state.cultural_center.name}</h3>
+        <h3>Current user: {username}</h3>
+        <h3>{cultural_center.name}</h3>
+        <CreateTourForm
+          onCreateTour={this.props.onCreateTour}
+          staffUser={id}
+          culturalCenter={cultural_center.id}
```

Comment:
These are confusing prop names. I think you should change them to represent what they really are.

* `staffUser` -> `staffUserId`
* `culturalCenter` -> `culturalCenterId`